setup.cfg

```diff
@@ -28,11 +23,11 @@ classifiers =
 test_suite = runtests.run_tests
 
 [options]
-python_requires = >=3.6
+python_requires = >=3.8
 packages = find:
 include_package_data = true
 install_requires =
-    django>=2.2
+    django>=3.2
```

Comment:
When we drop django 4.0 support, wouldn't it make sense to restrict this here: `django>=3.2,!=4.0`

Reply:
i think it's ok. we don't support 4.0 but we don't need to prevent it's usage